spec/spec_tests/crud_spec.rb

```diff
@@ -6,16 +6,16 @@
 
     spec = Mongo::CRUD::Spec.new(file)
 
+    before do
+      unless spec.server_version_satisfied?(authorized_client)
+        skip 'Version requirement not satisfied'
+      end
+    end
+
     context(spec.description) do
 
       spec.tests.each do |test|
 
-        around do |example|
-          if spec.server_version_satisfied?(authorized_client)
-            example.run
```

Comment:
Shouldn't this line still exist somewhere?

Reply:
before/after and around hooks work differently in this regard.